src/app/booking/OOSPlannerSection.js

```diff
@@ -225,6 +273,51 @@ function useAwaitingPaidStatusBill(billIdInitial = null) {
   const { setBillId } = useBillPolling({ billIdInitial, onSuccess });
 }
 
+function FriendsBookings( {children, weekDates, bookings} ) {
```

Comment:
<img width="1191" alt="image" src="https://github.com/digital-brother/ooscca-fe/assets/43747573/192c8ddb-c22e-4af9-9cfe-140e55bcb8ea">

Reply:
<img width="1177" alt="image" src="https://github.com/digital-brother/ooscca-fe/assets/43747573/99f18151-7843-4d87-8b10-68e2bbba3312">

How should we display friend bookings in case, if all friends have no bookings?

Reply:
<img width="1183" alt="image" src="https://github.com/digital-brother/ooscca-fe/assets/43747573/8c0e7449-cbc5-4701-a775-86d6eccd70cb">

How should we display a case, if one friend has bookings, and another one has not?

Reply:
<img width="1174" alt="image" src="https://github.com/digital-brother/ooscca-fe/assets/43747573/0d0064a9-b680-4422-8194-52258197199c">
There is no possibility to see both Milly and Jacob friends

Reply:
{children} has a special meaning in react, I would not pass a variable named children to a component. Please read about React children prop.